recipes-kernel/linux/linux-imx_5.4.bb

```diff
@@ -17,16 +17,16 @@ LIC_FILES_CHKSUM = "file://COPYING;md5=bbea815ee2795b2f4230826c0c6b8814"
 
 DEPENDS += "lzop-native bc-native"
 
-SRCBRANCH = "imx_5.4.70_2.3.0"
-LOCALVERSION = "-2.3.0"
-SRCREV = "4f2631b022d843c1f2a5d34eae2fd98927a1a6c7"
+SRCBRANCH = "imx_5.4.94_2.2.0"
+LOCALVERSION = "-2.2.0"
+SRCREV = "739c2edfa2b2179fb1db65017830f2af8fd6d627"
```

Comment:
Where this branch is hosted?

Reply:
Whoops, it should be pointing to: https://github.com/Freescale/linux-fslc/tree/5.4-2.2.x-imx

I had an overriding `.bbappend` that meant I didn't end up testing these changes.

Reply:
@alistair23 `linux-imx` is maintained by NXP team, advances with their release schedule, and does not receive any upstream merges from the community.

If you need to use the NXP tree with latest stable korg patch level - please switch your `PREFERRED_PROVIDER_virtual/kernel` to target `linux-fslc-imx` package instead.

-- andrey

Reply:
Ah thanks, I didn't realise there was a different kernel for the fslc tree.